### PR TITLE
Improvement/update kubernetes version to 1 12

### PR DESCRIPTION
**Pull request by ramanduh**
1. Update the version of kubernetes to 1.12.6. Test:

```
# kubectl version --kubeconfig=/etc/kubernetes/admin.conf
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.4", GitCommit:"c27b913fddd1a6c480c229191a087698aa92f0b1", GitTreeState:"clean", BuildDate:"2019-02-28T13:37:52Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.6", GitCommit:"ab91afd7062d4240e95e51ac00a18bd58fddd365", GitTreeState:"clean", BuildDate:"2019-02-26T12:49:28Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```

2. Remove tini from salt-master container. 
Test before:
```
[root@bootstrap /]# ps fax 
  PID TTY      STAT   TIME COMMAND
 4342 pts/0    Ss     0:00 /bin/bash
 4355 pts/0    R+     0:00  \_ ps fax 
    1 ?        Ss     0:00 /tini -- /usr/bin/salt-master
    6 ?        S      0:00 /usr/bin/python /usr/bin/salt-master
   11 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   16 ?        Sl     0:00  \_ /usr/bin/python /usr/bin/salt-master
   17 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   20 ?        S      0:11  \_ /usr/bin/python /usr/bin/salt-master
   21 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   22 ?        Sl     0:00  |   \_ /usr/bin/python /usr/bin/salt-master
   23 ?        Sl     0:07  |   \_ /usr/bin/python /usr/bin/salt-master
   24 ?        Sl     0:05  |   \_ /usr/bin/python /usr/bin/salt-master
   31 ?        Sl     0:05  |   \_ /usr/bin/python /usr/bin/salt-master
   32 ?        Sl     0:04  |   \_ /usr/bin/python /usr/bin/salt-master
   33 ?        Sl     0:04  |   \_ /usr/bin/python /usr/bin/salt-master
   34 ?        Sl     0:05  \_ /usr/bin/python /usr/bin/salt-master
```

Test after:
```
[root@bootstrap /]# ps fax 
  PID TTY      STAT   TIME COMMAND
  622 pts/0    Ss     0:00 /bin/bash
  635 pts/0    R+     0:00  \_ ps fax 
    6 ?        Ss     0:00 /usr/bin/python /usr/bin/salt-master
   15 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   20 ?        Sl     0:00  \_ /usr/bin/python /usr/bin/salt-master
   22 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   24 ?        S      0:02  \_ /usr/bin/python /usr/bin/salt-master
   25 ?        S      0:00  \_ /usr/bin/python /usr/bin/salt-master
   26 ?        Sl     0:00  |   \_ /usr/bin/python /usr/bin/salt-master
   27 ?        Sl     0:10  |   \_ /usr/bin/python /usr/bin/salt-master
   28 ?        Sl     0:07  |   \_ /usr/bin/python /usr/bin/salt-master
   29 ?        Sl     0:06  |   \_ /usr/bin/python /usr/bin/salt-master
   36 ?        Sl     0:06  |   \_ /usr/bin/python /usr/bin/salt-master
   37 ?        Sl     0:07  |   \_ /usr/bin/python /usr/bin/salt-master
   38 ?        Sl     0:01  \_ /usr/bin/python /usr/bin/salt-master
    1 ?        Ss     0:00 /pause
```

### CODE REVIEW

**Review by NicolasT — DISMISSED** [review]

File: images/salt-master/Dockerfile [hunk 1]
```diff
@@ -4,14 +4,6 @@ MAINTAINER moonshot-platform <moonshot-platform@scality.com>
 
 # Versions to use
 ARG SALT_VERSION=2018.3.3
-ARG TINI_VERSION=v0.18.0
-
-# Install and check tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
-
-COPY source.key source.key
```
Comment (gdemonet) [inline images/salt-master/Dockerfile:13]: Would be nice to untrack this file as well ;)

Comment (ramanduh) [reply images/salt-master/Dockerfile:13]: Will do

**Comment by ramanduh** [comment]
@bert-e reset

**Review by gdemonet — APPROVED** [review]

**Comment by ramanduh** [comment]
@bert-e approve

**Review by alexandre-allard — APPROVED** [review]

**Pull request merged**